test/parsers/aircraftCommandParser/argumentValidator.spec.js

```diff
@@ -295,3 +296,55 @@ ava('.squawkValidator() returns string when passed invalid squawk', t => {
     result = squawkValidator(['1a11']);
     t.true(result === 'Invalid argument. Expected \'0000\'-\'7777\' for the transponder code.');
 });
+
+ava('.crossingValidator() returns a string when passed the wrong number of arguments', t => {
```

Comment:
this whole set of tests is very nice.  well done! 👍

Reply:
:heart_eyes: